pensieve/analysis.py

```diff
@@ -35,6 +37,18 @@ class Analysis:
         mmd.search_count,
     ]
 
+    # list of standard statistics to be computed
+    STANDARD_STATISTICS = [
+        BootstrapOneBranch.from_config(
+            {
+                "num_samples": 1000,
+                "summary_quantiles": (0.5, 0.61),
+                "metrics": ["active_hours"],
+                "branches": ["branch1", "branch2"],
+            }
+        )
+    ]
```

Comment:
The intention here is to make it relatively easy to load the config from a config file.